e2e-tests/portal/cypress/integration/journeys/checker/issued-specs-new/submit-MIN-deal-with-issued-facility/submit-MIN-deal-with-issued-facilities.spec.js

```diff
@@ -54,7 +57,7 @@ context('A maker issues facilities, submits to checker; checker submits deal to
     // maker adds Issued Facilities and submits deal for review by checker
     //---------------------------------------------------------------
 
-    cy.login({ ...MAKER_LOGIN });
+    cy.loginBANK1_MAKER1;
```

Comment:
isn't this `cy.login(loginBANK1_MAKER1)`  - or something like that? 
I can't see where the `cy.loginBANK1_MAKER1` is defined separately